README.md

```diff
@@ -0,0 +1,36 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio desktop version
+
+> This project is part of the Microverse Program in which is resquested to do a portifolio in a desktop version using responsiveness and media queries. This portifolio is where all of my projects are, as well as my skills, in which will be uploaded on it in the future. The structure of this project is basically a simple page in html and css file. It is also part of the sketelon of my future portifolio, but in a mobile and a desktop version.
+
+## Built With
+
+- HTML & CSS
+
+## Live Demo (if available)
+
+[Portfolio-Online Link](https://mirelapitt.github.io/)
+

```

Comment:
The link to your page is not working:

![image](https://user-images.githubusercontent.com/38088818/167391782-8b215d3a-ad1f-4544-8999-25b51205d5b8.png)

Kindly add a working link to your README.
If you don't know how to deploy your website on GitHub, please follow these steps:

Go to Settings:
![image](https://user-images.githubusercontent.com/38088818/167392265-d71f0390-d009-4117-957c-4697a49b5fb7.png)

Then, navigate to pages:
![image](https://user-images.githubusercontent.com/38088818/167392596-1f9aaa56-a0a1-44ec-8245-0c0e030d4d33.png)

Choose the branch you want to publish and save:
![image](https://user-images.githubusercontent.com/38088818/167392860-9c63c547-c224-4a50-991c-3b8a5f3feb47.png)

GitHub will then provide you with a link:
![image](https://user-images.githubusercontent.com/38088818/167393013-83acfeae-5e2f-473f-8152-b7e205b8ce6d.png)